content/docs/bdd/discovery-workshop.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Discovery workshop
+subtitle: The session where you do your exploration work
+---
+
+### What is a discovery workshop?
+
+A discovery workshop is a conversation where technical and business people collaborate to explore, discover and agree as much as they can about the desired behaviour for a [User Story](/docs/terms/user-story/).
+
+### How do you conduct a discovery workshop?
+
+There are several discovery workshop models, these are just a few:
+
+[**Example Mapping**](/docs/bdd/example-mapping/)
+
+Uses a pack of four separately colored index cards to map rules (a summary of constraints/acceptance criteria the team has agreed upon) to examples (illustrations/cases of the acceptance criteria)
+
+[**OOPSI Mapping (Outcomes, Outputs, Processes, Scenarios, Inputs)**](https://jennyjmar.com/2016/04/16/bdd-discovery-and-oopsi/)
+
+Similar to Example Mapping, uses separately colored Post-it Notes to map shared processes/relationships between outputs and scenarios.
+
+[**Feature Mapping**](https://johnfergusonsmart.com/feature-mapping-a-simpler-path-from-stories-to-executable-acceptance-criteria/)
+
+Also uses separately colored Post-it Notes.  The team picks a story from the backlog, identifies the actors involved, breaks the story down into tasks, and maps those tasks to specific examples.
+
+### When should you hold a discovery workshop?
+
+As late as possible before development on a new User Story begins, in order to prevent details from being lost.  Conducting a discovery workshop as late as possible also gives the team enough leg room to shift their plans should new details surface.
```

Comment:
```suggestion
As late as possible before development on a new User Story begins, in order to prevent details from being lost. Conducting a discovery workshop as late as possible also gives the team enough leg room to shift their plans should new details surface.
```